frontend/src/components/Notification/Notification.tsx

```diff
@@ -83,10 +88,14 @@ const Notification = () => {
           )}
         </button>
       }
+      onOpen={() => {
+        setIsNotifOpen(true);
```

Comment:
I believe the "onOpen" callback isn't being called immediately.

On Windows and Microsoft Edge, the narrator says "notifications button collapsed" (not expected), followed by "notifications button expanded" (expected) when I click the notif bell to expand.

Basically there seems to be some tiny period between the narrator starting to read the button, and the button's aria-expanded being changed

Reply:
Deployed new version, please test it and let me know if you have any more issues!